lte/gateway/c/core/oai/include/ngap_messages_types.h

```diff
@@ -197,11 +189,11 @@ enum Ngcause {
   NGAP_CSFB_TRIGGERED,
   NGAP_NAS_UE_NOT_AVAILABLE_FOR_PS,
   NGAP_USER_INACTIVITY
-};
+} n2cause;
```

Comment:
Could you rename to n2cause_e to indicate as enum

Reply:
Comment addressed.